apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/gqldocument.kt

```diff
@@ -44,9 +44,10 @@ fun builtinDefinitions() = definitionsFromResources("builtins.graphqls")
 fun linkDefinitions() = definitionsFromResources("link.graphqls")
 
 /**
- * Extra apollo specific definitions from https://specs.apollo.dev/kotlin_labs/v0.1
+ * Extra apollo specific definitions from https://specs.apollo.dev/kotlin_labs/<[version]>
  */
-fun apolloDefinitions() = definitionsFromResources("apollo.graphqls")
+fun apolloDefinitions() = apolloDefinitions("v0.1")
```

Comment:
Make Deprecated?

Reply:
Probably make `apolloDefinitions(version: String)` public if we want to keep offering that possibility. I'd be surprised if anyone is relying on this at the moment but it might be useful for some use cases?